portfolio/src/main/webapp/recommendations.html

```diff
@@ -7,53 +7,56 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div>
+
       <nav id="my-nav">
         <ul>
           <li style="float:left"><a href="/index.html">Home</a></li>
           <li style="float:right">
-            <button onclick="userLogin()">Login</button>
+            <button onclick="recommendation()" id="resume-button">User Activity</button>
```

Comment:
I'm still new at HTML/CSS but it seems that class would be better than id, since there's more than one element named "resume-button".